src/components/containers/landingSection.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import Slider from "react-slick";
+
+import LandingSectionOne from "../views/LandingSectionOne";
+import LandingSectionTwo from "../views/LandingSectionTwo";
+import * as articleActions from "../../actions/articleActions";
+
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+
+class LandingSection extends Component {
+  static propTypes = {
+    articles: PropTypes.array.isRequired,
+    actions: PropTypes.object.isRequired
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      slider: {
+        dots: true,
+        infinite: true,
+        speed: 500,
+        autoPlaySpeed: 3000,
+        slidesToShow: 1,
+        slidesToScroll: 1
+      }
+    };
+  }
+
+  componentDidMount() {
+    const { actions } = this.props;
+    actions.article.getArticles("landingSection", true);
+  }
+
+  render() {
+    const { articles } = this.props;
+    const { slider } = this.state;
+    const articlesList = [[], [], []];
```

Comment:
Why is this array declared this way?

Reply:
It's a list of lists. The idea is to output 3 articles in 3 separate slides. The slider is currently not working so I had left that to work on in the coming new PRs.